### PR TITLE
docs(connectors): added notes to AMPQ and SQS connectors

### PR DESCRIPTION
**Pull request by igpetrov**
## Description

docs(connectors): added notes to AMPQ and SQS connectors

## When should this change go live?

Anytime, preferably with the next build since we have several customer requests related to this.

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [x] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [x] This functionality is already available but undocumented.
- [x] This is a bug fix or security concern.

## PR Checklist

- [ ] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [ ] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Review by sbuettner — DISMISSED** [review]

File: docs/components/connectors/out-of-the-box-connectors/amazon-sqs.md [hunk 1]
```diff
@@ -91,6 +91,13 @@ There are two options to authenticate the Connector with AWS:
 
 <TabItem value='inbound'>
 
+:::note
+To ensure consistent and stable work of Amazon Simple Queue Service Connector, it is not recommended
```
Comment (sbuettner) [inline docs/components/connectors/out-of-the-box-connectors/amazon-sqs.md:95]:
```suggestion
To ensure consistent and stable behaviour of the Amazon Simple Queue Service Connector, it is not recommended
```
Maybe @christinaausley has a better idea for phrasing this.

Comment (christinaausley) [reply docs/components/connectors/out-of-the-box-connectors/amazon-sqs.md:95]: to maintain stable behavior from the Amazon SQS Connector, do not subscribe multiple Amazon SQS Connectors to the same queue.

WDYT?

Comment (igpetrov) [reply docs/components/connectors/out-of-the-box-connectors/amazon-sqs.md:95]: @christinaausley perfect!

Comment (christinaausley) [reply docs/components/connectors/out-of-the-box-connectors/amazon-sqs.md]: Adjusted in latest commit 👍

**Pull request merged**